package-lock.json

```diff
@@ -19713,16 +19713,6 @@
 						"yauzl": "^2.7.0"
 					},
 					"dependencies": {
-						"are-we-there-yet": {
```

Comment:
This is annoying, I got this multiple times as well 😞 . Let's not add this change in this PR.

Reply:
I didn't mean to commit that. Removed.